azerothcore/azerothcore-wotlk
# fix (core/logging): fixed wrong decription of error during server startup

The Logger was pointing to the wrong table. Text is missed not in `broadcast_text_locale`, but in `broadcast_text`.

## How to Test the Changes:

What we have now:
1. add new record with some ID to broadcast_text_locale table
2. start server and see error.
3. add record to broadcast_text with same ID - error will dissapear

After applying the fix :
![изображение](https://user-images.githubusercontent.com/25766571/148967726-07ccbcbd-65be-4779-a142-33cb55e0d4c1.png)
